tests/localnet/conftest.py

```diff
@@ -7,13 +7,13 @@
 
 
 @pytest.fixture(autouse=True)
-def algod_health_fast_timings(mocker: MockerFixture) -> None:  # noqa: ignore[PT004]
+def algod_health_fast_timings(mocker: MockerFixture) -> None:  # noqa: PT004: ignore[PT004]
```

Comment:
why were we suppressing these instead of renaming the function to meet the linter rule?